### ISSUE TITLE
Stop refreshing the stack view

### ISSUE THREAD

**Issue by flarejune** [x64dbg/x64dbg#2181]
If the debugger is not Paused,Stop refreshing the stack view,
These stack data are actually meaningless.

**Comment by mrexodia** [comment]
I would disagree. Often when 'running' a process is simply in a waiting state. In that case the stack view can be very useful because it gives you information about the current state.
